pkg/client/unversioned/request.go

```diff
@@ -641,6 +660,12 @@ func (r *Request) Stream() (io.ReadCloser, error) {
 // fn at most once. It will return an error if a problem occurred prior to connecting to the
 // server - the provided function is responsible for handling server errors.
 func (r *Request) request(fn func(*http.Request, *http.Response)) error {
+	//Metrics for total request latency
```

Comment:
I don't think this is right, now that I look at it again. You're timing all retries in one chunk. You should measure them individually, no?